packages/main/src/Avatar.ts

```diff
@@ -70,13 +70,27 @@ import "@ui5/webcomponents-icons/dist/employee.js";
 })
 /**
 * Fired on mouseup, space and enter if avatar is interactive
-*
+* <b>Note:</b> The event will not be fired if the <code>disabled</code>
+* property is set to <code>true</code>.
 * @event
 * @private
 * @since 1.0.0-rc.11
 */
 @event("click")
 class Avatar extends UI5Element implements ITabbable {
+	/**
+	 * Defines whether the component is disabled.
+	 * A disabled component can't be pressed or
+	 * focused, and it is not in the tab chain.
+	 *
+	 * @type {boolean}
+	 * @name sap.ui.webc.main.Avatar.prototype.disabled
+	 * @defaultvalue false
+	 * @public
+	 */
+	@property({ type: Boolean })
+	disabled!: boolean;
+
 	/**
 	 * Defines if the avatar is interactive (focusable and pressable).
```

Comment:
I suggest adding a note to the "interactive" prop description, that if disabled is set, it wont take effect.